docs/PLUGIN.md

```diff
@@ -1,19 +1,30 @@
 ---
-title: Plugin
+title: Plugins
 type: docs
 weight: 2
 ---
 
-The plugin provides the auth flows that keycloak uses for x509 (CAC) authentication as well as some of the surrounding registration flows.
+Keycloak plugins provide additional custom logic to our Keycloak deployment. Below is a table of the current implemented plugins and how to interact with them.
 
-One of the plugin capabilities is [group authentication](https://github.com/defenseunicorns/uds-identity-config/blob/main/src/plugin/src/main/java/com/defenseunicorns/uds/keycloak/plugin/authentication/RequireGroupAuthenticator.java). This capability allows for defining a keycloak group membership that is required to access an application. There is [an e2e test](https://github.com/defenseunicorns/uds-identity-config/blob/main/src/test/cypress/e2e/group-authz.cy.ts) that demonstrates grafana being deployed and requiring that users accessing its dashboard be members of the `/UDS Core/Admin` group. There is [additional documentation](https://github.com/defenseunicorns/uds-core/blob/v0.23.0/docs/configuration/uds-operator.md?plain=1#L23-L26) for configuring an application for group authentication in uds-core.
+## Current Plugins
+| Name | Type | Description |
```

Comment:
It might be good to add an additional column and categorize them as `Authenticator || Mapper || Event Listener`?